test/optimizations/remoteValueForwarding/bharshbarg/alias.chpl

```diff
@@ -0,0 +1,31 @@
+
+var g : int = 0;
+
+var gr : R;
+
+record R {
+  var x : int;
+}
```

Comment:
How about making a corresponding version using a class and a global reference to to a class instance, to test the code I was just pointing out?

e.g.

class C { ... }
var instance = new C(...);
ref globalRef = instance;

... testClass(globalRef) ...

proc testClass(const ref globalRef) {
 ...
 globalRef.x += 1;
 ...
}